deleteReading.php

```diff
@@ -0,0 +1,9 @@
+<? 
+include('config.php'); 
+$device_id = $_GET['device_id']; 
+$time = $_GET['time']; 
+mysql_query("DELETE FROM `reading` WHERE `device_id` = $device_id AND `time` = $time") ; 
```

Comment:
There's something weird going on with `DATETIME` equality. The row doesn't actually get deleted.